src/Security/Security.Test/ScenarioTests/TestController.cs

```diff
@@ -63,21 +63,33 @@ public void RunPowerShellTest(ServiceManagement.Common.Models.XunitTracingInterc
                     _helper.RMProfileModule,
                     _helper.GetRMModulePath(@"AzureRM.Security.psd1"),
                     "ScenarioTests\\Common.ps1",
-                    "ScenarioTests\\" + callingClassName + ".ps1");
+                    "ScenarioTests\\" + callingClassName + ".ps1",
+                    _helper.GetRMModulePath(@"AzureRM.Resources"),
```

Comment:
Instead of using the Resources and Storage modules, please use our internal version of these cmdlets.  This is just changing this import to the two ps1 files: https://github.com/Azure/azure-powershell/blob/d99d9f8f8d730782c5f654243e55138385fd9fe1/src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/TestsBase.cs#L145, and updating the client to the common version: https://github.com/Azure/azure-powershell/blob/d99d9f8f8d730782c5f654243e55138385fd9fe1/src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/TestsBase.cs#L32, https://github.com/Azure/azure-powershell/blob/d99d9f8f8d730782c5f654243e55138385fd9fe1/src/RecoveryServices/RecoveryServices.Backup.Test/ScenarioTests/TestsBase.cs#L29